src/explore.js

```diff
@@ -24,8 +24,10 @@ export const exploreSetup = () => {
 };
 
 export const exploreResponse = (type, message) => {
-  chrome.tabs.query({ active: true, url: 'https://*/explore*' }, (tabs) => {
-    chrome.tabs.sendMessage(tabs[0].id, { type, message });
+  chrome.tabs.query({ active: true, url: exploreHost }, (tabs) => {
+    if (tabs.length > 0) {
+      chrome.tabs.sendMessage(tabs[0].id, { type, message });
+    }
```

Comment:
Ya, I don't really know how you would alert the frontend of an error here. I just added this so that, if you navigate to a different tab while explore is running, there won't be a console error and it won't try to send a message when it can't.